src/validators/url.py

```diff
@@ -29,6 +29,8 @@ def _path_regex():
     return re.compile(
         # allowed symbols
         r"^[\/a-z0-9\-\.\_\~\!\$\&\'\(\)\*\+\,\;\=\:\@\%"
+        # symbols / pictographs
+        + r"\U0001F300-\U0001F5FF"
         # emoticons / emoji
         + r"\U0001F600-\U0001F64F"
```

Comment:
These ranges overlap. You could simply have:

```diff
- # symbols / pictographs
- r"\U0001F300-\U0001F5FF"
- # emoticons / emoji
- r"\U0001F600-\U0001F64F"
+ # symbols / pictographs / emojis / emoticons
+ r"\U0001F300-\U0001F5FF"
```

Reply:
Also, please rebase `prousso:chore/add-symbols-pictographs-support` on `master`, when you're done.

Reply:
Do you mean
```suggestion
+ # symbols / pictographs / emojis / emoticons
+ + r"\U0001F300-\U0001F64F"
```
?

Reply:
Sorry, my bad keep them separate.

ref: https://unicodeplus.com/plane/1